binary.json

```diff
@@ -0,0 +1,54 @@
+{
+   "cases": [
```

Comment:
This could probably be called "decimal" to reflect that there will probably be a `decimal` function/method being used in the solution.